libs/pages/services/src/lib/feature/page-application-create-feature/page-application-install-feature/page-application-install-feature.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  ApplicationRequest,
+  BuildModeEnum,
+  BuildPackLanguageEnum,
+  ContainerRequest,
+  PortProtocolEnum,
+} from 'qovery-typescript-axios'
+import { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useNavigate, useParams } from 'react-router'
+import { createApplication, postApplicationActionsDeploy } from '@qovery/domains/application'
+import { selectAllRepository } from '@qovery/domains/organization'
+import { MemorySizeEnum, ServiceTypeEnum } from '@qovery/shared/enums'
+import {
+  SERVICES_APPLICATION_CREATION_URL,
+  SERVICES_CREATION_GENERAL_URL,
+  SERVICES_CREATION_PORTS_URL,
+  SERVICES_CREATION_RESOURCES_URL,
+  SERVICES_URL,
+} from '@qovery/shared/router'
+import { FunnelFlowBody } from '@qovery/shared/ui'
+import { buildGitRepoUrl, convertCpuToVCpu } from '@qovery/shared/utils'
+import { AppDispatch } from '@qovery/store/data'
+import PageApplicationInstall from '../../../ui/page-application-create/page-application-install/page-application-install'
+import { useApplicationContainerCreateContext } from '../page-application-create-feature'
+
+export function PageApplicationInstallFeature() {
```

Comment:
I think the naming is weird here, it's not an "install" because you post data only, wdyt about "post" ?